app/Observers/Gpao/ManufacturingOrderObserver.php

```
@@ -71,17 +74,19 @@ protected function updateMachineUsage(ManufacturingOrder $order): void
 
             if ($machine->usage_hours >= $machine->maintenance_interval_hours) {
                 $hasOpenTicket = MachineMaintenanceTicket::where('machine_id', $machine->id)
-                    ->where('type', 'preventive')
-                    ->where('status', 'open')
+                    ->where('type', MachineMaintenanceTicketType::PREVENTIVE)
+                    ->where('status', MachineMaintenanceTicketStatus::OPEN)
                     ->exists();
 
                 if (! $hasOpenTicket) {
-                    MachineMaintenanceTicket::create([
+                    $ticket = MachineMaintenanceTicket::create([
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Gpao\MachineMaintenanceTicket::create().


(staticMethod.notFound)